legal-api/src/legal_api/models/alias.py

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module holds data for aliases."""
+from __future__ import annotations
+
+from enum import Enum
+
+from .db import db
+
+
+class Alias(db.Model):  # pylint: disable=too-many-instance-attributes
+    """This class manages the aliases."""
+
+    class AliasType(Enum):
+        """Render an Enum of the types of aliases."""
+
+        TRANSLATION = 'TRANSLATION'
+        DBA = 'DBA'
+
+    __tablename__ = 'aliases'
+
```

Comment:
It needs to be versioned
 __versioned__ = {}

Reply:
Updated